client/my-sites/activity/filterbar/style.scss

```diff
@@ -392,7 +393,7 @@
 			width: 24px;
 		}
 
-		@include break-xlarge {
+		@include breakpoint-deprecated( ">660px" ) {
```

Comment:
Using this deprecated breakpoint to ensure everything tweaks at the same viewport.